packages/shared/src/utils/route/route.ts

```diff
@@ -1,6 +1,7 @@
 // Checks if pathname matches route. (Works even with query string /?)
 // TODO: Add test cases for this
 type TRoute = {
+    component?: React.ComponentType<any>;
```

Comment:
A temporary solution, as we can't write `component?: React.ComponentType | typeof Redirect;` because `Redirect` (from _react-router-dom_) isn't in the package's dependencies